### PR TITLE
firewall rules page accessibility fixes

### PR DESCRIPTION
**Pull request by jfayre**
Added aria-label attribute to buttons on firewall rules table. Also removes aria-hidden attribute from modal dialogs. 
Using aria-hidden makes dialogs invisible to screen readers.

### CODE REVIEW

File: src/www/firewall_rules.php [hunk 1]
```diff
@@ -366,7 +366,12 @@ function filter_rule_item_alias_tooltip($alias_name)
                     $("#action").val("del");
                     $("#iform").submit()
                 }
-              }]
+              }],
+              // remove aria-hidden
+              onshown: function(dialogRef){
+                  dialogRef.getModal().removeAttr('aria-hidden');
```
Comment (AdSchellevis) [inline src/www/firewall_rules.php:372]: I think we should remove this in the bootstrap dialog itself.

https://github.com/opnsense/core/blob/6355a3debd3052526f26c03c029de6de6ed84f2f/src/opnsense/www/js/bootstrap-dialog.js#L392

(By the way, as https://github.com/nakupanda/bootstrap3-dialog seems to be stale, we can update the source on our end.)

I think the original bootstrap 3 examples contained the `aria-hidden` at some time, which might explain why a lot of these forms contain them.

**Review by AdSchellevis — CHANGES_REQUESTED** [review]
If you remove the `onshow` events, we can merge this as well

**Comment by jfayre** [comment]
Awesome! I really wasn’t looking forward to updating all of these dialogs. The onshown events should be removed now.

File: src/www/firewall_rules.php [hunk 2]
```diff
@@ -367,6 +367,7 @@ function filter_rule_item_alias_tooltip($alias_name)
                     $("#iform").submit()
                 }
               }]
+
```
Comment (AdSchellevis) [inline src/www/firewall_rules.php:370]: can you remove the extra line break as well? thanks

File: src/www/firewall_rules.php [hunk 3]
```diff
@@ -386,6 +387,7 @@ function filter_rule_item_alias_tooltip($alias_name)
...
```
Comment (AdSchellevis) [inline src/www/firewall_rules.php:390]: same here

**Comment by AdSchellevis** [comment]
@jfayre thanks!

**Pull request merged**